ceph.spec.in

```diff
@@ -248,6 +248,12 @@ on commodity hardware and delivers object, block and file system storage.
 Summary:       Ceph Base Package
 Group:         System Environment/Base
 Requires:      ceph-common = %{epoch}:%{version}-%{release}
+# redhat-lsb-core/lsb-release for lsb_release executable
+%if 0{?suse_version}
+Requires:      lsb-release
+%else
+Requires:      redhat-lsb-core
```

Comment:
Please drop the `%else` part. We explicitly don't want `redhat-lsb-core` because it pulls in a lot of other things, bloating Docker images that use Ceph.